EvolutionStrategies/PythonDraft/Population.py

```diff
@@ -0,0 +1,133 @@
+import numpy
+from math import cos, pi, sqrt, exp
+from matplotlib import pyplot
+
+POPULATION = {'names': ('a', 'b', 'c', 'std_dev_a', 'std_dev_b', 'std_dev_c'),
+              'formats': ('float64', 'float64', 'float64', 'float64', 'float64', 'float64')}
+
+
+def open_file_and_process_data(file_name):
+    file = numpy.loadtxt(file_name, dtype={'names': ('we', 'wy'), 'formats': ('float64', 'float64')})
+    return file
+
+
+class Population:
+    def __init__(self, size, file_data, stop_criteria):
+        self.parent_size = size
+        self.offspring_size = size * 6
+        self.file_data = file_data
+        self.stop_criteria = stop_criteria
+        self.iterations = 0
+        self.print_index = 0
+        self.population_vector = numpy.empty(self.parent_size, dtype=POPULATION)
+        self.offspring_population = numpy.empty([self.offspring_size], dtype=POPULATION)
+        self.col_names = ['a', 'b', 'c', 'std_dev_a', 'std_dev_b', 'std_dev_c']
+
+    def initialize(self, min_value, max_value, std_dev):
+        for i in range(len(self.col_names[0:3])):
```

Comment:
change this loop to something pythonic like for col in cols[0:3] and make another for that second case